diff --git a/dist/index.mjs b/dist/index.mjs
index c421da0e95452e80b8fa74c02af601dc85337bde..37eff454ce1b722acda9b7fdf2f8be8f6d96c099 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -274,7 +274,7 @@ var ScalarRender = (version, config, cdn) => `<!doctype html>
 </html>`;
 
 // src/utils.ts
-import path from "path";
+import path from "node:path";
 
 // node_modules/@sinclair/typebox/build/esm/type/symbols/symbols.mjs
 var TransformKind = Symbol.for("TypeBox.Transform");
