diff --git a/dist/index.mjs b/dist/index.mjs
index 5b75602818c90215aca94704a0756918653e4b6b..b080b95d6a4129aa7244493d671fa6f2b8055523 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -326,7 +326,7 @@ var ScalarRender = (info, version, config, cdn) => `<!doctype html>
 </html>`;
 
 // src/utils.ts
-import { normalize } from "pathe";
+import { normalize } from "node:pathe";
 
 // node_modules/@sinclair/typebox/build/esm/type/symbols/symbols.mjs
 var TransformKind = Symbol.for("TypeBox.Transform");
