diff --git a/dist/types/Text.js b/dist/types/Text.js
index e939e019d0826a52c94bcc54023c6db648daf6df..7ce6a0233e46a3fe88be7749cff69df715cdc903 100644
--- a/dist/types/Text.js
+++ b/dist/types/Text.js
@@ -7,17 +7,13 @@ const useResyncValue_1 = require("./useResyncValue");
 const Input = react_native_theming_1.styled.TextInput(({ theme }) => ({
     ...(0, common_1.inputStyle)(theme),
 }));
-const TextType = ({ arg, onChange, isPristine }) => {
+const TextType = ({ arg = {}, onChange = (value) => value, isPristine }) => {
     const { setCurrentValue, key } = (0, useResyncValue_1.useResyncValue)(arg.value, isPristine);
     return ((0, jsx_runtime_1.jsx)(Input, { testID: arg.name, defaultValue: arg.value, onChangeText: (text) => {
             onChange(text);
             setCurrentValue(text);
         }, autoCapitalize: "none", underlineColorAndroid: "transparent" }, key));
 };
-TextType.defaultProps = {
-    arg: {},
-    onChange: (value) => value,
-};
 TextType.propTypes = {};
 TextType.serialize = (value) => value;
 TextType.deserialize = (value) => value;
