diff --git a/dist/index.mjs b/dist/index.mjs
index c3ecabed5b776a6f061d35876976c0ac8f8c9691..2e91aa472c466fb630c20d54afe95b72fc3c9dd9 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -3013,7 +3013,10 @@ var cloneInference = (inference) => ({
   set: inference.set,
   server: inference.server
 });
-var redirect = (url, status = 302) => Response.redirect(url, status);
+var redirect = (url, status = 302) => {
+  const _res = Response.redirect(url, status);
+  return new Response(_res.body, _res);
+};
 var ELYSIA_FORM_DATA = Symbol("ElysiaFormData");
 var ELYSIA_REQUEST_ID = Symbol("ElysiaRequestId");
 var form = (items) => {
diff --git a/dist/utils.mjs b/dist/utils.mjs
index c44a5d1b088ce4b5d88f687c884d4744c85dd26f..55afce1e51ef35fcb09892cf0b5bda134ed61f37 100644
--- a/dist/utils.mjs
+++ b/dist/utils.mjs
@@ -1636,7 +1636,10 @@ var cloneInference = (inference) => ({
   set: inference.set,
   server: inference.server
 });
-var redirect = (url, status = 302) => Response.redirect(url, status);
+var redirect = (url, status = 302) => {
+  const _res = Response.redirect(url, status);
+  return new Response(_res.body, _res);
+}
 var ELYSIA_FORM_DATA = Symbol("ElysiaFormData");
 var ELYSIA_REQUEST_ID = Symbol("ElysiaRequestId");
 var form = (items) => {
